File: WebSolution/TFLOBL/Entities/Task.cs
namespace Transflower.TFLPortal.TFLOBL.Entities;
public class Task
{
   public int Id{get;set;}
   public string? Title{get;set;}
   public string? Description{get;set;}
   public DateTime AssignDate{get;set;}
   public DateTime StartDate{get;set;}
   public DateTime DueDate{get;set;}
   public int AssignedTo{get;set;}
   public int ProjectId{get;set;}
   public string? Status{get;set;}
}

File: Solution/UISolution/Draft/src/app/shared/models/task.ts
export class Task {
    constructor(
      public taskId: number,
      public title: string,
      public tasktype: string,
      public description: string,
      public projectId: number,
      public sprintId: number,
      public assignedTo: number,
      public assignedBy: number,
      public createdDate :string ,
      public assignedDate :string ,
      public startDate :string,
      public dueDate: string,
      public status :string
    ) {}
  }